src/CredentialManagement/CredentialManagement.csproj

```diff
@@ -14,7 +14,7 @@
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
     <TargetFrameworkProfile />
-    <BuildType Condition="Exists('..\..\script\src\ApiClientConfiguration.cs')">Internal</BuildType>
+    <BuildType Condition="Exists('..\..\script\src\ApiClientConfiguration_User.cs')">Internal</BuildType>
```

Comment:
Just a observation having seen all the .csproj tweaks. It would be nice to factor out these common signing related properties into a separate .props file. I'll do this in a separate PR along with the public key stuff.